File: code/app/src/main/java/com/example/avtokampi_android_app/models/Slika.kt
package com.example.avtokampi_android_app.models

import java.sql.Blob

data class Slika(
    val id: Int,
    val slika: Blob,
    val createdAt: String,
    val updatedAt: String,
    val uporabnik: Uporabnik,
    val avtokamp: Avtokamp
)

File: avtokampi/lib/models/Slika.dart
class Slika {
    int id;
    String slika;
    String createdAt;
    String updatedAt;
    int avtokamp;

    Slika.nov();

    Slika(this.id, this.slika, this.createdAt, this.updatedAt,
        this.avtokamp);

    static Slika fromJson(json) {
        Slika s = new Slika.nov();
        s.id = json['cenikId'];
        s.slika = json['slika'];
        s.createdAt = json['createdAt'];
        s.updatedAt = json['updatedAt'];
        s.avtokamp = json['avtokamp'];
        return s;
    }
}
